tasks/managed/rh-sign-image/rh-sign-image.yaml

```diff
@@ -247,8 +247,10 @@ spec:
             RAW_OUTPUT=$(skopeo inspect --retry-times 3 --no-tags --raw "docker://${referenceContainerImage}")
             # Always sign the top level sha
             manifest_digests="${referenceContainerImage#*@}"
-            # For multi arch, also sign all the manifests inside
-            if [ "$(jq -r '.mediaType' <<< "$RAW_OUTPUT")" != "application/vnd.oci.image.manifest.v1+json" ] ; then
+            # For multi arch and index images, also sign all the manifests inside
+            MEDIATYPE="$(jq -r '.mediaType' <<< "$RAW_OUTPUT")"
+            if [ "$MEDIATYPE" == "application/vnd.oci.image.index.v1+json" ] || \
+               [ "$MEDIATYPE" == "application/vnd.docker.distribution.manifest.list.v2+json" ]; then
```

Comment:
Do we have any test cases for other situations where we check for the Docker v2s2 media types?

Reply:
We haven't gotten past this, so I can't confirm nor deny whether other, later steps in the release pipeline would fail.

Reply:
One thought: Would it make it potentially safer to just add your type to the list of non-multi arch types, instead of switching the logic to list the multi arch types?
Meaning:
```
            if [ "$MEDIATYPE" != "application/vnd.oci.image.manifest.v1+json" ] && \
               [ "$MEDIATYPE" != "application/vnd.docker.distribution.manifest.v2+json" ]; then
```

Reply:
Updated. Thanks for the review!